pkg/authstore/store.go

```diff
@@ -0,0 +1,13 @@
+package authstore
+
+import "github.com/benchkram/bob/bobauth"
+
+type Store interface {
```

Comment:
Does this has multiple implementations? If not I would get rid of it to avoid interface pollution. Also by name and number of methods it looks to me like something very concrete, not really a generic behaviour.